### DISCUSSION TITLE
GUI Problem

### DISCUSSION THREAD

**Discussion by heiyu8** [rgthree/rgthree-comfy#314] [Q&A]
Hello author, how do I murder these damn ICONS?
![6efdc2cee3a75c1f56c9508a8c5d760](https://github.com/user-attachments/assets/e5a353ce-96f0-4a20-a6da-28558f4603db)

**Comment by rgthree** [comment]
Hmm.. looks like you're either missing a CSS file from loading (rgthree.css) or, more likely, another extension is overwriting the CSS that styles the icons. Are you using a translation extension or something else that could be hacking around in the menu?

**Comment by andreszs** [comment]
Never seen that before, you should first disable the rgthree extension and restart the GUI, to check if this is really the right plugin to blame.
